frontend/src/index.ejs

```diff
@@ -13,6 +13,13 @@
 
 <body>
   <div id="App"></div>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
```

Comment:
@bcherny what's the impact/point of adding all of these scripts?

Reply:
So it fixes an immediate issue, but it also has some broader pros/cons. The immediate issue is that chartjs-plugin-stacked100 assumes that [`Chart` is defined globally](https://github.com/y-takey/chartjs-plugin-stacked100/blob/09174207bbab825c16e0538d56671c9d112a7b6a/src/index.js#L103), when in fact it's not because we are compiling to CommonJS. The line you commented on makes sure it's defined in the global namespace, and that when we `import abc from 'chart.js'`, it actually defers to the global version (this is configured in webpack.config.js).

The broader pros/cons of using these scripts is:

Pros:

- Faster build times (because we no longer need to concat all of these scripts into bundle.js)
- Dev builds are more prod-like (we were already doing this for prod builds)

Cons:

- You now need an internet connection to run the frontend locally